apps/app/src/app/pages/(components)/components/(tabs)/tabs--vertical.preview.ts

```diff
@@ -48,13 +40,13 @@ import { HlmTabsContentDirective, HlmTabsListDirective, HlmTabsTriggerDirective
 		class: 'block w-full max-w-lg min-h-[400px]',
 	},
 	template: `
-		<brn-tabs class="mx-auto flex max-w-3xl flex-row space-x-2" orientation="vertical" value="account">
-			<brn-tabs-list hlmTabsList orientation="vertical" aria-label="tabs example">
-				<button class="w-full" hlmTabsTrigger brnTabsTrigger="account">Account</button>
-				<button class="w-full" hlmTabsTrigger brnTabsTrigger="password">Password</button>
-				<button class="w-full" hlmTabsTrigger brnTabsTrigger="danger">Danger Zone</button>
-			</brn-tabs-list>
-			<div hlmTabsContent brnTabsContent="account">
+		<div brnTabs="account" class="mx-auto flex max-w-3xl flex-row space-x-2" orientation="vertical">
+			<div hlmTabsList hlm orientation="vertical" aria-label="tabs example">
```

Comment:
How do you feel about turning this into <hlm-tabs-list>? 
I think that would be even cleaner!

Reply:
Oh sorry I did not see your comment. 
What do you mean by "turning  into ?"